<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>724</width>
    <height>603</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>samplebrain 0.0.3</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="controlTab">
       <attribute name="title">
        <string>search</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_15">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_7">
          <item>
           <widget class="QLabel" name="label_19">
            <property name="font">
             <font>
              <family>Comic Sans MS</family>
              <pointsize>20</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>tweakage</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_12">
            <item>
             <widget class="QPushButton" name="pushButtonPlay">
              <property name="font">
               <font>
                <family>Comic Sans MS</family>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>Start</string>
              </property>
              <property name="flat">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButtonStop">
              <property name="font">
               <font>
                <family>Comic Sans MS</family>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>Stop</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_16">
              <item>
               <widget class="QLabel" name="label_6">
                <property name="font">
                 <font>
                  <family>Comic Sans MS</family>
                  <pointsize>14</pointsize>
                  <weight>75</weight>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="text">
                 <string>fft &lt;-&gt; mfcc ratio</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QDial" name="dialRatio">
                <property name="enabled">
                 <bool>true</bool>
                </property>
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>60</width>
                  <height>60</height>
                 </size>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="doubleSpinBoxRatio"/>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_4">
              <item>
               <widget class="QLabel" name="label_7">
                <property name="font">
                 <font>
                  <family>Comic Sans MS</family>
                  <pointsize>14</pointsize>
                  <weight>75</weight>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="text">
                 <string>fft subsection 1</string>
                </property>
               </widget>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_3">
                <item>
                 <widget class="QLabel" name="label_9">
                  <property name="font">
                   <font>
                    <family>Comic Sans MS</family>
                    <weight>75</weight>
                    <bold>true</bold>
                   </font>
                  </property>
                  <property name="text">
                   <string>Start</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QSpinBox" name="spinBoxFFT1Start"/>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_9">
                <item>
                 <widget class="QLabel" name="label_10">
                  <property name="font">
                   <font>
                    <family>Comic Sans MS</family>
                    <weight>75</weight>
                    <bold>true</bold>
                   </font>
                  </property>
                  <property name="text">
                   <string>End</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QSpinBox" name="spinBoxFFT1End"/>
                </item>
               </layout>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_5">
              <item>
               <widget class="QLabel" name="label_8">
                <property name="font">
                 <font>
                  <family>Comic Sans MS</family>
                  <pointsize>14</pointsize>
                  <weight>75</weight>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="text">
                 <string>fft subsection 2</string>
                </property>
               </widget>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_10">
                <item>
                 <widget class="QLabel" name="label_11">
                  <property name="font">
                   <font>
                    <family>Comic Sans MS</family>
                    <weight>75</weight>
                    <bold>true</bold>
                   </font>
                  </property>
                  <property name="text">
                   <string>Start</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QSpinBox" name="spinBoxFFT2Start"/>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_11">
                <item>
                 <widget class="QLabel" name="label_12">
                  <property name="font">
                   <font>
                    <family>Comic Sans MS</family>
                    <weight>75</weight>
                    <bold>true</bold>
                   </font>
                  </property>
                  <property name="text">
                   <string>End</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QSpinBox" name="spinBoxFFT2End"/>
                </item>
               </layout>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="verticalSpacer_3">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>323</width>
              <height>511</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_6">
            <item>
             <widget class="QLabel" name="label_13">
              <property name="font">
               <font>
                <family>Comic Sans MS</family>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>Volume</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDial" name="dialVolume">
              <property name="value">
               <number>99</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_9">
          <item>
           <widget class="QLabel" name="label_16">
            <property name="font">
             <font>
              <family>Comic Sans MS</family>
              <pointsize>20</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>target sound</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_15">
            <property name="font">
             <font>
              <family>Comic Sans MS</family>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>no sound yet...</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonLoadTarget">
            <property name="font">
             <font>
              <family>Comic Sans MS</family>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>load target</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_13">
            <item>
             <widget class="QLabel" name="label_17">
              <property name="font">
               <font>
                <family>Comic Sans MS</family>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>block size</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="spinBoxBlockSizeTarget"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_14">
            <item>
             <widget class="QLabel" name="label_18">
              <property name="font">
               <font>
                <family>Comic Sans MS</family>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>block overlap</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="spinBoxBlockOverlapTarget"/>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonGenerateTarget">
            <property name="font">
             <font>
              <family>Comic Sans MS</family>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>generate blocks</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="sampleTab">
       <attribute name="title">
        <string>brain</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="font">
             <font>
              <family>Comic Sans MS</family>
              <pointsize>20</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>brain parameters</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QLabel" name="label">
              <property name="font">
               <font>
                <family>Comic Sans MS</family>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>block size</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="spinBoxBlockSize"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item>
             <widget class="QLabel" name="label_2">
              <property name="font">
               <font>
                <family>Comic Sans MS</family>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>block overlap</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="spinBoxBlockOverlap"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QLabel" name="label_4">
              <property name="font">
               <font>
                <family>Comic Sans MS</family>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>fft spectrum size</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="spinBoxSpectSize"/>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonGenerate">
            <property name="font">
             <font>
              <family>Comic Sans MS</family>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>generate</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_7">
            <item>
             <widget class="QPushButton" name="pushButtonLosdBrain">
              <property name="font">
               <font>
                <family>Comic Sans MS</family>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>load</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButtonSaveBrain">
              <property name="font">
               <font>
                <family>Comic Sans MS</family>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>save</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QLabel" name="label_5">
            <property name="font">
             <font>
              <family>Comic Sans MS</family>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>brain contents</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QListWidget" name="listWidgetSounds"/>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QPushButton" name="pushButtonLoadSound">
              <property name="font">
               <font>
                <family>Comic Sans MS</family>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>load sound</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButtonPlay</sender>
   <signal>released()</signal>
   <receiver>MainWindow</receiver>
   <slot>play_slot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>64</x>
     <y>62</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>301</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonStop</sender>
   <signal>released()</signal>
   <receiver>MainWindow</receiver>
   <slot>stop_slot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>155</x>
     <y>62</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>301</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dialRatio</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>ratio_slot(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>109</x>
     <y>154</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>301</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>doubleSpinBoxRatio</sender>
   <signal>valueChanged(double)</signal>
   <receiver>MainWindow</receiver>
   <slot>ratio_slot(double)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>109</x>
     <y>223</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>301</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBoxFFT1Start</sender>
   <signal>valueChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>fft1_start_slot(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>154</x>
     <y>277</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>301</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBoxFFT1End</sender>
   <signal>valueChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>fft1_end_slot(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>154</x>
     <y>310</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>301</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBoxFFT2Start</sender>
   <signal>valueChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>fft2_start_slot(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>154</x>
     <y>366</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>301</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBoxFFT2End</sender>
   <signal>valueChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>fft2_end_slot(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>154</x>
     <y>399</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>301</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonLoadTarget</sender>
   <signal>released()</signal>
   <receiver>MainWindow</receiver>
   <slot>load_target()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>277</x>
     <y>103</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>301</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBoxBlockSizeTarget</sender>
   <signal>valueChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>target_block_size(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>313</x>
     <y>136</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>301</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBoxBlockOverlapTarget</sender>
   <signal>valueChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>target_block_overlap(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>320</x>
     <y>169</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>301</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonGenerateTarget</sender>
   <signal>released()</signal>
   <receiver>MainWindow</receiver>
   <slot>generate_target_blocks()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>277</x>
     <y>202</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>301</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBoxBlockSize</sender>
   <signal>valueChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>block_size(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>188</x>
     <y>105</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>301</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBoxBlockOverlap</sender>
   <signal>valueChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>block_overlap(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>188</x>
     <y>138</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>301</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBoxSpectSize</sender>
   <signal>valueChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>fft_spectrum_size(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>188</x>
     <y>171</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>301</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonGenerate</sender>
   <signal>released()</signal>
   <receiver>MainWindow</receiver>
   <slot>generate()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>131</x>
     <y>204</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>301</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonLoadSound</sender>
   <signal>released()</signal>
   <receiver>MainWindow</receiver>
   <slot>load_sound()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>381</x>
     <y>544</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>301</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dialVolume</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>volume_slot(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>189</x>
     <y>480</y>
    </hint>
    <hint type="destinationlabel">
     <x>361</x>
     <y>301</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>play_slot()</slot>
  <slot>stop_slot()</slot>
  <slot>ratio_slot(double)</slot>
  <slot>ratio_slot(int)</slot>
  <slot>fft1_start_slot(int)</slot>
  <slot>fft2_start_slot(int)</slot>
  <slot>fft1_end_slot(int)</slot>
  <slot>fft2_end_slot(int)</slot>
  <slot>volume_slot(int)</slot>
  <slot>run_slot()</slot>
  <slot>block_size(int)</slot>
  <slot>block_overlap(int)</slot>
  <slot>fft_spectrum_size(int)</slot>
  <slot>generate()</slot>
  <slot>load_sound()</slot>
  <slot>delete_sound()</slot>
  <slot>slot2()</slot>
  <slot>load_target()</slot>
  <slot>target_block_size(int)</slot>
  <slot>target_block_overlap(int)</slot>
  <slot>generate_target_blocks()</slot>
 </slots>
</ui>
